docs/notary/setup-Komodo-Notary-Node.md

```diff
@@ -568,18 +580,70 @@ sudo ln -sf /home/$USER/hush3/src/hushd /usr/local/bin/hushd
 ```bash
 cd ~
 git clone https://github.com/jl777/chips3 -b dev
-cd chips3
+cd chips3 
+```
+
+#### Step 2: Build
+
+##### Ubuntu 18.04
+
+```bash
 ./build.sh
 ```
 
-#### Step 2: Symlink the compiled binaries
+##### Debian 10
+
+Replace the contents of the `build.sh` file with the following code
+
+```bash
+#!/bin/bash
+
+berkeleydb() {
+    CHIPS_ROOT=$(pwd)
+    CHIPS_PREFIX="${CHIPS_ROOT}/db4"
+    mkdir -p $CHIPS_PREFIX
+    wget -N 'http://download.oracle.com/berkeley-db/db-4.8.30.NC.tar.gz'
+    echo '12edc0df75bf9abd7f82f821795bcee50f42cb2e5f76a6a281b85732798364ef db-4.8.30.NC.tar.gz' | sha256sum -c
+    tar -xzvf db-4.8.30.NC.tar.gz
+    if [ -f /etc/debian_version ]; then
+        DEBIAN_VERSION=$(cat /etc/debian_version)
+        DEBIAN_VERSION=${DEBIAN_VERSION%.*}
+        if [ "$DEBIAN_VERSION" -eq 10 ]; then
+            #https://www.fsanmartin.co/compiling-berkeley-db-4-8-30-in-ubuntu-19/
+            sed -i 's/__atomic_compare_exchange/__atomic_compare_exchange_db/g' db-4.8.30.NC/dbinc/atomic.h
+        fi
+    fi
+    cd db-4.8.30.NC/build_unix/
+    ../dist/configure -enable-cxx -disable-shared -with-pic -prefix=$CHIPS_PREFIX
+    make install
+    cd $CHIPS_ROOT
+}
+
+buildCHIPS() {
+    git pull
+    ./autogen.sh
+    ./configure LDFLAGS="-L${CHIPS_PREFIX}/lib/" CPPFLAGS="-I${CHIPS_PREFIX}/include/" --with-gui=no --disable-tests --disable-bench --without-miniupnpc --enable-experimental-asm --enable-static --disable-shared
+    make -j$(nproc)
+}
+berkeleydb
+buildCHIPS
+echo "Done building CHIPS!"
+```
+
+then, run the script
+
+```bash
+./build.sh
```

Comment:
if you want to be precise - you'll need make script executable for the user: `chmod u+x build.sh` to execute then

Reply:
above the script, I am asking the user to replace the contents of the existing build.sh file. so, this is ok I think